### DISCUSSION TITLE
Token Sorting

### DISCUSSION THREAD

**Discussion by calvinogood** [six7/figma-tokens#1096] [Q&A] [answered]
<img width="400" alt="Screenshot 2022-07-29 at 10 30 22 AM" src="https://user-images.githubusercontent.com/2776021/181670531-f345996e-7593-4fee-a1cf-2694c52f3cba.png">

Hi, how can I perform the sorting here? Is it possible?

**Comment by rbosker** [comment, marked as answer]
Hi @calvinogood , currently its not possible to sort this list of tokens. I created a feature request for it: https://github.com/six7/figma-tokens/issues/1099
Feel free to elaborate/upvote the ticket
